plugins/aea-ledger-cosmos/aea_ledger_cosmos/cosmos.py

```diff
@@ -1020,72 +960,111 @@ def get_transfer_transaction(  # pylint: disable=arguments-differ
         :param gas: the gas used.
         :param memo: memo to include in tx.
         :param chain_id: the chain ID of the transaction.
+        :param account_number: Account number
+        :param sequence: Sequence
         :param kwargs: keyword arguments.
         :return: the transfer transaction
         """
         denom = denom if denom is not None else self.denom
+
         chain_id = chain_id if chain_id is not None else self.chain_id
-        account_number, sequence = self._try_get_account_number_and_sequence(
-            sender_address
-        )
+
         if account_number is None or sequence is None:
-            return None  # pragma: nocover
-        transfer_msg = {
-            "type": "cosmos-sdk/MsgSend",
-            "value": {
-                "amount": [{"amount": str(amount), "denom": denom}],
-                "from_address": sender_address,
-                "to_address": destination_address,
-            },
-        }
+            account_number, sequence = self._try_get_account_number_and_sequence(
+                sender_address
+            )
+            if account_number is None or sequence is None:
+                return None  # pragma: nocover
+
+        tx_fee_coins = [Coin(denom=denom, amount=str(tx_fee))]
+        amount_coins = [Coin(denom=denom, amount=str(amount))]
```

Comment:
There is some discrepancy between this file and `_cosmos.py` in the fetchai plugin, which is in this method until this point. Is this diff intentional? I don't see how these two files may not be identical.